app/controllers/users_controller.rb

```diff
@@ -0,0 +1,55 @@
+class UsersController < ApplicationController
+  include ApplicationHelper
+
+  def index # login page
+    @user = User.new
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create # POST
+    @user = User.new(user_params)
+    if @user.save
+      session[:user_id] = @user.id
+      redirect_to user_url(@user)
+    else
+      flash[:error] = @user.errors.full_messages
+      redirect_to new_user_path
```

Comment:
If you redirect here the form will clear. It would be kinder to the user to render :new